.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - "node"
```

Comment:
how does this work? does it run both linting and and ava tests?

did you have to sign up for a service to get this to work?